cluster.py

```diff
@@ -1,21 +1,46 @@
 import logging
+import socket
+import time
 from pathlib import Path
-from typing import Dict
+from typing import Dict, Tuple
 
 from ccmlib import scylla_cluster as ccm
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
 
+def get_cluster_id() -> Tuple[socket.socket, int]:
+    """gets integer number that will be used for ip prefix of cluster nodes,
+    so it's possible to run tests in parallel.
+
+    Returns tuple of socket and cluster id, where socket later needs to be closed."""
+    logger.info("Getting cluster id for support parallel tests...")
+    sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    for cluster_id in range(126):
+        try:
+            sock.bind((f'127.0.{cluster_id}.1', 48783))  # random port
```

Comment:
`id` is always expected to be unique.
In your case it is reusable thing.
So, wrong naming. 

Maybe `subnet_suffix` ?

Reply:
I refactored it to show the meaning. Actually it is not about cluster id, rather making no other cluster will use the same ip range. So I changed it to `ip_prefix`